__tests__/client/prerender.spec.js

```diff
@@ -134,15 +133,15 @@ describe('loadPrerenderScripts', () => {
 describe('moveHelmetScripts', () => {
   beforeEach(() => {
     jest.clearAllMocks();
-    [...document.querySelectorAll('script')].forEach((scriptTag) => { scriptTag.parentElement.removeChild(scriptTag); });
+    [...document.querySelectorAll('script')].forEach((scriptTag) => { scriptTag.remove(); });
 
     jest.spyOn(document, 'addEventListener').mockImplementation(() => { /* noop */ });
   });
 
   const createScript = ({ src, helmet }) => {
     const script = document.createElement('script');
     script.src = src;
-    if (helmet) script.setAttribute('data-react-helmet', true);
+    if (helmet) script.dataset.reactHelmet = true;
```

Comment:
why did this change?

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v23.0.0/docs/rules/prefer-dataset.md